java/src/test/java/eu/solven/cleanthat/engine/java/refactorer/cases/do_not_format_me/TestUnnecessaryModifierCases.java

```diff
@@ -224,6 +224,19 @@ public class MyInnerClass {
 			// public and static are redundant
 			public static enum MyEnum {
 			}
+
+			// final is redundant
+			private static final void privateStaticFinalMethod() {
+			}
+
+			// final is not redundant
+			// https://stackoverflow.com/questions/1743715/behaviour-of-final-static-method/1743748#1743748
```

Comment:
I preferably put these comments in the mutators, not in the unittests. No worry, I'll move them later.